tests/rhtap-demo/README.md

```diff
@@ -59,7 +59,7 @@ ginkgo --label-filter='verify-stage' ./cmd/
 Steps to run 'rhtap-demos':
 
 1) Follow the instructions from the [Readme](../../docs/Installation.md) scripts to install AppStudio in e2e mode
-2) Run the e2e suite: `./bin/e2e-appstudio --ginkgo.label-filter="rhtap-demo"`
+2) Run the e2e suite: `./bin/e2e-appstudio --ginkgo.label-filter="konflux"`
```

Comment:
will it be good to rename `rhtap-demo` directory to `konflux` as well ... i.e. `tests/konflux-demo/konflux-demo.go`?

Reply:
Then do we need to have another ticket that deals with renaming the test file to `konflux-demo.go`?

Reply:
I've made the renaming already https://github.com/kasemAlem/e2e-tests/tree/KONFLUX-3150/tests/konflux-demo